### ISSUE TITLE
"new URL()" parses magnet links

### ISSUE THREAD

**Issue by limitedeternity** [whatwg/url#403]
Title says all. Is that intentional?

![screenshot](https://user-images.githubusercontent.com/24318966/42941513-a53ed7dc-8b65-11e8-85a5-96472afc116e.png)

**Comment by domenic** [comment]
Why wouldn't it? They're URLs...

**Comment by limitedeternity** [comment]
It parses incorrectly anyway. 
Btw, just a few days ago "new URL()" didn't parse these links. Is that about regex changes or something else?

**Comment by domenic** [comment]
It sounds like you're talking about a specific implementation---from your screenshot, Chrome?---and not about the standard itself. There have been no recent changes to URL parsing, as you can see from https://github.com/whatwg/url/commits/master

**Comment by limitedeternity** [comment]
Yep, I see. This behavior was just quite unexpected for me, because I used "new URL()" for link validation (lol) and it showed better results than any regular expression I found on the net. Also, implementation is useful when you are trying to construct full url from relative one. So when I passed a magnet link to it as a test case I expected "new URL()" to throw a TypeError, but it didn't and that's why I thought "was that made intentionally?". 
Thanks for your time anyway. Closing issue ^^

P.s: Yes, that's Chrome.
